src/en/Readhive.lua

```diff
@@ -0,0 +1,143 @@
+-- {"id":4304,"ver":"1.0.0","libVer":"1.0.0","author":"MechTechnology"}
+
+local baseURL = "https://readhive.org"
+
+local json = Require("dkjson")
+
+local text = function(v)
+	return v:text()
+end
+
+
+local function shrinkURL(url)
+	return url:gsub("^.-readhive%.org", "")
+end
+
+local function expandURL(url)
+	return baseURL .. url
+end
+
+local function clearNewLines(v)
+	-- Remove addtional empty paragraphs if they exist
+	local toRemove = {}
+	v:traverse(NodeVisitor(function(v)
+		if v:tagName() == "p" and v:text() == "" then
+			toRemove[#toRemove+1] = v
+		end
+		if v:hasAttr("border") then
+			v:removeAttr("border")
+		end
+	end, nil, true))
+	for _,v in pairs(toRemove) do
+		v:remove()
+	end
+	return v
+end
+
+local function getLatestListing(data)
+	local page = data[PAGE]
+	local url = "/page/" ..page.. "/?"
```

Comment:
any reason for the empty query string (and useless temporary variables)?

Reply:
yes... my bad, fixed that